recipes/coverage/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set version = "3.7.1" %}
+{% set md5 = "c47b36ceb17eaff3ecfab3bcd347d0df" %}
+{% set pkgname = "coverage" %}
+
+package:
+  name: {{ pkgname }}
+  version: {{ version }}
+
+source:
+  fn: {{ pkgname }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ pkgname[0] }}/{{ pkgname }}/{{ pkgname }}-{{ version }}.tar.gz
+  md5: {{ md5 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
```

Comment:
hmm, but I thought convention (as with the other package recipes I submitted) was that the script was supposed to be a single script, rather than a set of `yaml`, `bat` and `sh` scripts?